reconcile/utils/openshift_resource.py

```diff
@@ -140,7 +142,6 @@ def ignorable_field(val):
             "generation",
             "selfLink",
             "uid",
-            "status",
```

Comment:
oh i see. they key point is this here. we ignored status fields whereever they were defined. but what we wanted to do was only ignore it when it was the regular `status` section of a K8S resource.